x-pack/plugins/apm/public/components/app/correlations/latency_correlations.tsx

```diff
@@ -198,7 +166,8 @@ export function LatencyCorrelations({ onFilter }: { onFilter: () => void }) {
           'xpack.apm.correlations.latencyCorrelations.correlationsTable.fieldValueLabel',
           { defaultMessage: 'Field value' }
         ),
-        render: (fieldValue: string) => String(fieldValue).slice(0, 50),
+        render: (fieldValue: FieldValuePair['fieldValue']) =>
```

Comment:
Use the second argument to avoid manual typings, and thereby getting better type safety.

```suggestion
        render: (_, { fieldValue }) =>
```

Reply:
(same for the rest of the table

Reply:
👍 - changed for all similar callbacks in 787d9d7.